microreactors/drum_rotation/hpc_tests

```diff
@@ -0,0 +1,74 @@
+[Tests]
+  [drum_mesh]
+    type = RunApp
+    input = drum.i
+    cli_args = '--mesh-only'
+    executable_pattern = 'griffin*'
+    group = griffin_performance
+  []
+  [fine_mesh]
+    type = RunApp
+    input = empire_2d_CD_fine.i
+    cli_args = '--mesh-only'
+    executable_pattern = 'griffin*'
+    prereq = drum_mesh
+    group = griffin_performance
+  []
+  [coarse_mesh]
+    type = RunApp
+    input = empire_2d_CD_coarse.i
+    cli_args = '--mesh-only'
+    executable_pattern = 'griffin*'
+    group = 'griffin_performance'
+  []
+  # this test is primarily meant to help keep track of Griffin run times
+  [neutronics_eigenvalue]
+    type = CSVDiff
+    input = neutronics_eigenvalue.i
+    cli_args = "Executioner/richardson_rel_tol=1e-6 Outputs/csv=true
+                Outputs/exodus=false
+                Outputs/file_base=neutronics_eigenvalue_heavy
+                TransportSystems/sn/NPolar=3
```

Comment:
I changed these two params to 2 and 6 in my PR. I guess that should be enough for a performance test. If my PR goes in first, we will not need the new gold files. Or remove these two params here and the new gold file. I certainly can change this after it is merged, but thought that we may want to have a slightly cleaner history.